styles/Canonical/400-Enforce-inclusive-terms.yml

```
@@ -1,5 +1,6 @@
 extends: existence
 message: "Replace '%s' with an inclusive term"
+scope: summary
```

My Vale is a bit rusty, but `summary` excludes headings and table entries. Any reason not to reuse the scopes from other rules?

```suggestion
scope:
  - sentence
  - heading
  - table
```